src/v3/src/components/AuthenticatorButton/AuthenticatorButton.tsx

```diff
@@ -140,7 +145,13 @@ const AuthenticatorButton: UISchemaElementComponent<{
         <Typography
           variant="h3"
           id={`${iconName}-label`}
-          sx={{ fontSize: '1rem', margin: 0, textAlign: 'start' }}
+          textAlign='left'
```

Comment:
Will specific directions such as text align left (or margin right) cause problems for right-to-left languages?

Reply:
CSS logical properties cannot be supported in IE11, the plan is to use [rtl plugins](https://mui.com/material-ui/guides/right-to-left/#3-install-the-rtl-plugin) to handle related issues.

Also see draft PR: https://github.com/okta/okta-signin-widget/pull/3283